CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/CRFHeartRateStepViewController.swift

```diff
@@ -0,0 +1,254 @@
+//
+//  CRFHeartRateStepViewController.swift
+//  CardiorespiratoryFitness
+//
+//  Copyright © 2017-2018 Sage Bionetworks. All rights reserved.
```

Comment:
just 2018? or were these files maybe moved from another repo and that's why Github sees them as "new"?